libs/gi/page-home/src/QuickLinksCard.tsx

```diff
@@ -124,45 +104,63 @@ export default function QuickLinksCard() {
         avatar={<InsertLink fontSize="large" />}
       />
       <Divider />
-      <CardContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
-        {buttons.map((btnProps, i) => {
-          const { title, icon, tooltip, color } = btnProps
-          let button
-          if ('to' in btnProps)
-            button = (
-              <Button
-                fullWidth
-                key={i}
-                color={color}
-                component={RouterLink}
-                to={btnProps.to}
-                startIcon={icon}
-              >
-                {title(t)}
-              </Button>
-            )
-          if ('url' in btnProps)
-            button = (
-              <Button
-                fullWidth
-                key={i}
-                color={color}
-                component={Link}
-                href={btnProps.url}
-                target="_blank"
-                rel="noopener"
-                startIcon={icon}
-              >
-                {title(t)}
-              </Button>
-            )
-          if (!button) return null
-          return (
-            <Tooltip key={i} title={tooltip(t)} placement="top" arrow>
-              {button}
-            </Tooltip>
-          )
-        })}
+      <CardContent
+        sx={{ display: 'grid', gridTemplateColumns: 'repeat(2,1fr)', gap: 1 }}
+      >
+        <Box sx={{ border: 1, borderRadius: 4, padding: 2 }}>
```

Comment:
Can you try converting these cards into actual cards, instead of outlined divs?
```suggestion
        <CardThemed bgt="light>
          <CardContent>
```

Reply:
Aha showing my rusty MUI here. Absolutely, let me look into that

Reply:
![image](https://github.com/frzyc/genshin-optimizer/assets/25755821/f15f5a0d-b0ac-4c7d-ad72-b282ac8465d6)
Great suggestion! Think that looks really nice and segmetns the areas really well.